src/plugins/dragresize_ie11.js

```diff
@@ -485,9 +485,16 @@
 				if (
 					editor.filter.checkFeature(this.features.dimension) &&
 					editor.config.ae_dragresize_ie11_disableResizer !== true
-				)
+				) {
 					setupResizer(this);
 
+					var dragHandlerStyle = this.dragHandlerContainer.$.style;
```

Comment:
(nit) Can we please make this a `const`?

Reply:
Fixed